docs/src/InvariantTheory/intro.md

```diff
@@ -65,3 +65,12 @@ and the survey article
 - [DJ98](@cite)
 
 provide details on theory and algorithms as well as references.
+
+
+## Contact
+
+Please direct questions about this part of OSCAR to the following people:
+* [Wolfram Decker](https://www.mathematik.uni-kl.de/agag/personen/leitung/decker/seite),
+* [Max Horn](https://www.mathematik.uni-kl.de/en/agag/people/head/prof-dr-max-horn/seite).
```

Comment:
The `/seite` bit can be safely dropped from all these links

Reply:
And I think currently it is a mix of German and English versions (at least for the uni-kl pages)

Reply:
I removed (or at least tried) to remove all `/seite` pieces from the uni-kl links.

@lgoe-ac thank you for the pointer. I double checked this too. All but one link should now direct to an English page.

The one exception is the website of @afkafkafk13 (https://uol.de/anne-fruehbis-krueger). This page only offers me German as language. Then again, maybe it is sufficient to find the email address (top left, grey box)?

Reply:
I will enquire, whether an englisch version can be provided. For some odd reason, this is not standard at UOL. (According to their policy, Webpages are provided by administration, not be the individual).